### PR TITLE
docs: add how to set up cluster info from webpage to our repo

### PR DESCRIPTION
**Pull request by FaGru3n**
#### What would be changed or will be added with this PR?
- move and adapt format for Markdown files for our internal guides to onboarding tasks
- last remaining `docs/how-to-setup-aks-cluster-via-terraform.md`  from catena-x ng website
- remove specific docusaurus content and tranfer it to Markdown format like https://docusaurus.io/docs/next/markdown-features/admonitions

#### Why do i want to change this?
- reason https://github.com/eclipse-tractusx/sig-infra/issues/348 reduce content on webpage

#### Validation
- check the new files and its base content on our webpage https://catenax-ng.github.io/docs/internal/how-to-setup-aks-cluster-via-terraform

### CODE REVIEW

File: docs/how-to-setup-aks-cluster-via-terraform.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+az ad sp create-for-rbac --skip-assignment
+```
+
+You'll need the 'client id' and 'secret id' values later on.
+
+## Terraform state
+
+Terraform uses [`state`](https://www.terraform.io/language/state) to track status and differences of your real world
+resources and your configuration. While being very useful in most situation, we actively decided to NOT manage/save the
+terraform state to track our AKS cluster resources. We think, that sharing and storing the state file is too much of an
```
Comment (SebastianBezold) [inline docs/how-to-setup-aks-cluster-via-terraform.md:34]: This is definitely outdated and we should point to the backend configuration in the terraform scripts

**Review by SebastianBezold — CHANGES_REQUESTED** [review]
Just the part about the state should be updated. Rest is fine

File: docs/how-to-setup-aks-cluster-via-terraform.md (hunk 1 above)
Comment (FaGru3n) [reply docs/how-to-setup-aks-cluster-via-terraform.md]: [22e0f6c](https://github.com/catenax-ng/cloud-infra/pull/14/commits/22e0f6c489ea4f8c5e8cd3311b541296a5085708)

Did some changes, where directly you want to point to the scripts?

Comment (SebastianBezold) [reply docs/how-to-setup-aks-cluster-via-terraform.md]: This is where it is defined for the core cluster for example: https://github.com/catenax-ng/cloud-infra/blob/main/01_core_cluster/providers.tf#L2

Comment (SebastianBezold) [reply docs/how-to-setup-aks-cluster-via-terraform.md]: But fine for now. At least the outdated parts are removed. We can improve the docs with up-to-date stuff later on

**Review by SebastianBezold — APPROVED** [review]

**Pull request merged**